build_tools/benchmarks/shark/run_report.sh

```diff
@@ -27,16 +27,15 @@ pip install --upgrade pip
 pip install pandas
 pip install jinja2
 
-export SHARK_VERSION=`ls ${RAW_RESULTS_DIR} | grep "cpu_shark" | sed -e "s/^cpu_shark_\(.*\).csv$/\1/g"`
-export IREE_VERSION=`ls ${RAW_RESULTS_DIR} | grep "cpu_iree" | sed -e "s/^cpu_iree_\(.*\).csv$/\1/g"`
+SHARK_VERSION=$(ls ${RAW_RESULTS_DIR} | grep "cpu_shark" | sed -e "s/^cpu_shark_\(.*\).csv$/\1/g")
```

Comment:
Parsing `ls` is generally a bad idea. It's intended for human consumption. In this case it's not even the slightly more stable option that prints one filename per line. What are you actually trying to do here? Extract the version info from a file name? Can we pass the version info more directly rather than trying to parse it back out of file names? If not, I still think parsing `ls` here is not necessary

```suggestion
SHARK_VERSION=$(basename "${RAW_RESULTS_DIR}"/cpu_shark* | sed -e "s/^cpu_shark_\(.*\).csv$/\1/g")
```

Reply:
The version info isn't actually isn't needed anymore. I've updated it to not include this and instead look for the files directly.